content/04.network_applications.md

```diff
@@ -45,10 +45,10 @@ Other efforts have used variants of this algorithm to construct their own low di
 Typically, eigenmaps work well when knowledge graphs have a sparse number of edges between nodes but struggle when presented with denser networks [@doi:10.1093/bioinformatics/btz718; @doi:10.1371/journal.pcbi.1005621; @arxiv:1905.09763].
 An open area of exploration is to adapt these methods to accommodate knowledge graphs that have a large number of edges.
 
-Matrix factorization is a powerful technique that uses a matrices such as  an adjacency matrix as input.
+Matrix factorization is a powerful technique that uses matrices such as an adjacency matrix as input.
```

Comment:
Is there a context in which you'd use this where you wouldn't be factorizing the adjacency matrix? I think you could simplify this sentence if not.